docs/source/technical_tutorials/search/catalog.rst

```diff
@@ -69,7 +69,7 @@ For all NASA MAAP users, access to NASA'S Operational data is provided via a fed
 
 Anyone can access data through Earthdata Login as well.
 
-Find more documentation about how to access data in CMR in the `Access <accessing.html>`_ section of this documentation.
+Find more documentation about how to access data in CMR in the `Access </docs/source/technical_tutorials/access>`_ section of this documentation.
```

Comment:
I think `/docs/source/technical_tutorials/accessing.html` works, does `/docs/source/technical_tutorials/access` work as well?

Reply:
Hmm, let me see what /docs/source/technical_tutorials/accessing.html does, but /docs/source/technical_tutorials/access works for bringing you to the access folder

Reply:
Yeah if I change it to /docs/source/technical_tutorials/accessing.html then it takes me to a 404 error page and says it isn't found

Reply:
oh, interesting 🤔 if you've got a local build going, do you want to show me?

Reply:
Sure!